Proposal-eng.md

```diff
@@ -0,0 +1,64 @@
+# Proposal - UIC HCC Computer Society
+
+Dear Sir / Madam:
+
+
+
+We are computer technology enthusiasts in UIC from different majors and different grades, who founded UIC HCC Computer Society (Unofficial) in October, 2016, in hope of spreading the knowledge of computer science and technology, conveying the sprit of open source, and sharing as well as focusing on the frontier development of computer science. Today, we wish to establish UIC HCC Computer Society officially under Innovation Center at the moment when UIC New Campus has moved smoothly and Innovation Center will be established soon. We hope to make contributions to students and staffs in UIC as well as the Industry-University Integration plan of UIC with the support of this platform.
+
+This proposal will introduce overview, vision, mission, task and expected achievement of UIC HCC Computer Society (hereafter this text will be abbreviated as Society). In addition, we will list our expectations to Innovation Center and UIC.
+
+
+
+## Society Overview
+
+UIC HCC Computer Society established unofficially in October, 2016, and have 20 members from different majors and different grades (members list in appendix). We make our target **Bridging You And Technology** as our responsibility, and spread knowledge related to computer science to UIC teachers and students in different occasions actively. With the emerge of new technologies in computer science, we find it difficult to achieve our target and keep our development under the unofficial operation mode. Therefore, our society wish to obtain the strong support from UIC and Innovation Center to carry out more science literacy, practice and research activities.
+
+Currently, we have our public website, GitHub Organization, mail system and WeChat official account.
+
+
+
+## Vision of Society
+
+Originated from the fundamental mission and vision of Society, **Bridging You And Technology**, our society hopes to carry out **popularizaion of knowledge related to computer science**, **software and hardware development** cooperated with our school or society organizations, **participation on open source projects** and **researching on frontier theories of computer science and their application**.
```

Comment:
1. suggest to use "fundamental objective or goal"
2. suggest to replace "originated from" with "In order to implement a fulfilling practice on the fundamental.... "
3. better add a "the" before each Society
4. suggest use sentence structure like"In order to.......of the Society that Bridging You And Technology,"
5. could better be "popularisation of computer science related knowledge"
6. participation in
7. research on frontier.....

Reply:
Will all fix.